### PR TITLE
Deletes the Beach Biodome lavaland ruin

### PR DESCRIPTION
**Pull request by genessee596**
**THE WHAT**
This PR completely removes the beach biodome ruin on lavaland, and with it the Beach Bros and Beach Bartender ghost roles. I tested this out on a private server, and it _shouldn't_ runtime despite not changing any other files that mention the beach biodome. Go figure!

**THE HOW**
Literally just deleting 'lavaland_biodome_beach.dmm'. That's it.

**THE WHY**
This is more about the ghost roles instead of about the ruin itself. Ghost roles are supposed to do one of two things; add to the round in some minor way (such as syndicate comms agents misdirecting people) or to give whoever's playing the ghost role some sort of tutorial or practice area on a specific game mechanic. (Such as Charlie Station teaching engineering, fighting simplemobs, and maneuvering throughout space; or the Lifebringers letting players practice botany in relative peace) Beach Bros do neither of these. Firstly, they very rarely to never actually contribute to the round, only occasionally heading to the station to validhunt, nor do they teach anything to their players (_maybe_ beach bartender might teach bartending, but that's pushing it - and still doesn't explain the standard beach bro). Their only use in games, at this point, is for players to spawn in as to run to the station (or ash lizards) and murderbone.

TL;DR: Beach Bros have no reason to exist besides metagrudging.

## Changelog
:cl:
del: Removed the Beach Biodome lavaland ruin, and the Beach Bro, Beach Bartender and Beach Lifeguard ghost roles with it.
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Nah brah.

https://www.youtube.com/watch?v=p13yZAjhU0M

**Comment by trexter555** [comment]
if you think its a bad ruin then why not try making some changes to it instead of outright deleting it

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Bruh you didn't even try
image

**Comment by Cyberboss** [comment]
Can we get more dislikes than YT rewind 2019?

**Comment by carshalash** [comment]
This grommet had a wipeout and wants to take away the waves.

**Comment by FancyCyclops** [comment]
Inevitably merged when the outrage comes, because everything they hate is good for the game.

**Comment by kriskog** [comment]
Has anyone actually spawned in this ruin, and stuck around to roleplay with the noone else that has either?

**Comment by nightred** [comment]
I did the bartender once, i did the beach bum twice, and did the rp.
Never using it again.

**Comment by cacogen** [comment]
It sucks for anything but a chance to get back to the station. Not that I'm saying you should remove it.

**Comment by HomocidlePancakes** [comment]
Literally only exists for people to tide in to station. Fuck it.

**Comment by dr-bee** [comment]
They cant even be used for roleplay because 3 out of the 4 spawners cant even speak common. Literally a waste of a ruin roll. I will be glad to see them go.

**Comment by itseasytosee** [comment]
I can't wait for Beach Biodome 2 surfin' space-SA

**Comment by wesoda25** [comment]
I agree its useless, but you should replace it with something nice to replace it. Whoever removed stranded vets did well with it - simple but nice replacement

**Comment by ArcaneMusic** [comment]
Or, add some food supplies to the map and make it a kind kitchen/bartender tutorial to try out making some exotic foods.

**Comment by cacogen** [comment]
^ That's a good idea. The seed vault is fun because you can easily grow cool things, and because of lack of internals you can't leave until you grow the appropriate plant to offset the oxygen damage.

**Comment by evilcyborgmrt** [comment]
I also support turning this into a chef/bar training area. How about just removing the internals and pickaxes from the beach zone?

On a side note, it would be nice if there were some indication for new players that we have all these ghost roles where they can practice mechanics without having to worry about the station burning down around them.

**Comment by Shadowflame909** [comment]
Doesn't the syndicate base already fulfill the niche of bar/kitchen. Plus. There are so many of those bar/kitchen ruins that another one would just be bloat.

I mean for Ian's sake. Even miners can buy capsules that literally create Bars.

**Comment by Daxxed** [comment]
Ashlizards love to find these, they have a bunch of sunglasses, materials to craft machines, medkits and extra lives if a poor soul decides to take the spawn.

Other than that, you are completely right.
My "best" encounter with these was when one spawned to steal the last legion tendril and then nearly kill me with the meathook he got from it.
When that failed, he respawned 3 more times to try it with spears instead.

Occasionally you'll have the peaceful one just trying to get to station but I will make it my personal duty to take them back to where they belong.

**Comment by CAPTTLasky** [comment]
> 
> 
> Doesn't the syndicate base already fulfill the niche of bar/kitchen. Plus. There are so many of those bar/kitchen ruins that another one would just be bloat.
> 
> I mean for Ian's sake. Even miners can buy capsules that literally create Bars.

You can't call the Syndicate base's kitchen a real kitchen, come on.

**Comment by cacogen** [comment]
Don't think I've ever seen anyone on the syndicate base make food. It's rare to see them even do chemistry/research?/grenade making. Me personally I just like comms agent

**Comment by Farquaar** [comment]
Why not just add a bit of description saying something like "You're on Spring Break, brah. You shouldn't leave the beachdome unless you're in serious danger!"

Bam. Done.

**Comment by Shadowflame909** [comment]
it'd save admins from banning some good players. Because that'd be a mistake just like the ash lizard situation.

To summarize, when ash lizard role text was change. People didn't notice the change because they've played that role for hundreds and hundreds of rounds. Not everyone follows github and all that.  So they get warned or banned because they have some sort of similar ghost role note, or they went on a literal murderbone spree as they would have before the ghost role change.

So, **Changing the flavor-text will just get people banned, and have the role ultimately see less use.**

Lets not do that, because everything it offers can still be had via other ruins or items. 

Removing it is the best outcome.

**Comment by CAPTTLasky** [comment]
I'm making a revamp version as we speak, PR will be up later today.

**Comment by MCterra10** [comment]
@evilcyborgmrt I think keeping the internals stuff would be good, because it was fun playing bartending and heading out mining for minerals to make rare drinks

**Comment by ExcessiveUseOfCobblestone** [comment]
do it properly

**Comment by CAPTTLasky** [comment]
Revamped Beach Biodome
#48548
